images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -487,6 +487,11 @@ function Get-PowershellVersion {
     return $powershellVersion
 }
 
+function Get-SwigVersion {
+    $swigVersion = Run-Command "swig -version"
+    return "Swig $swigVersion"
```

Comment:
should it be either `swig` as it's for windows or `SWIG` as it's in the official docs?

Reply:
Fixed. Swig - so as not to break the code style